dashboard.css

```diff
@@ -0,0 +1,123 @@
+.main {
+  width: 100%;
+  max-width: 50rem;
+  margin: 2rem auto;
+  margin-bottom: 5rem;
+  padding: 1rem;
+  border-radius: 1rem;
+}
+
+.dashboard {
+  display: flex;
+  gap: 2rem;
+}
+
+.dashboard__avatar.avatar img {
+  border-radius: 1rem;
+  border: none;
+}
+.dashboard__info {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.info__progress-bar {
+  height: 0.5rem;
+  border-radius: 0.5rem;
+  width: 100%;
+  background: var(--primary-color-light);
+}
+.progress-bar__bg {
+  background: var(--primary-color-dark);
+  height: 100%;
+  width: 75%;
+  border-radius: 0.5rem;
+}
+
+.info__activity-items {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+.activity__item {
+  display: flex;
+  align-items: flex-end;
+  gap: 0.5rem;
+}
+
+.stat__number {
+  font-size: 1.5rem;
+  font-weight: bold;
+}
+.stat__title {
+  font-size: 0.75rem;
+  color: var(--grey-color);
+}
+.featured {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  margin-top: 2rem;
+  gap: 1rem;
+}
+.categories-section__title,
+.league-board__title {
+  margin-bottom: 1rem;
+}
+.categories-wrapper {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 1rem;
+}
+.category {
+  position: relative;
+  transition: all 0.2s ease-in-out;
+  cursor: pointer;
+}
+.category:hover {
+  transform: scale(1.05);
+}
+.category .category__title {
+  position: absolute;
+  width: 100%;
+  text-align: center;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  color: var(--white-color);
+  padding: 0.5rem;
+  background: rgba(0, 0, 0, 0.5);
```

Comment:
Should use variables instead of direct values.

Reply:
I am using it only here. So I don't think I need to set it as a variable.